HW03 Maria Evers.Rmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "HW03"
+author: "Maria Evers"
+date: "7/20/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## R Markdown
+
+This is an R Markdown document. Markdown is a simple formatting syntax for authoring HTML, PDF, and MS Word documents. For more details on using R Markdown see <http://rmarkdown.rstudio.com>.
+
+When you click the **Knit** button a document will be generated that includes both content as well as the output of any embedded R code chunks within the document. You can embed an R code chunk like this:
+
+```{r cars}
+summary(cars)
+```
+
+## Including Plots
+
+You can also embed plots, for example:
+
+```{r pressure, echo=FALSE}
+plot(pressure)
+```
+
+Note that the `echo = FALSE` parameter was added to the code chunk to prevent printing of the R code that generated the plot.
+
+
+
+
+## Homework 3: Graphing your own data
+### I will be graphing data from an intracellular ATP assay performed on H1299 NSCLC cells treated with various concentrations of ATP
+### Cancer cells are able to take up ATP from their extracellular environment via a process called **macropinocytosis**, significantly increasing intracellular ATP levels 
+#### The ATP assay utilized uses a firefly luciferase system to measure the amount of ATP present in biological samples, in this case H1299 cell lysate. Luminescence is measured and is directly proportional to the amount of ATP present. 
```

Comment:
Really like your intro!! And great idea of trying ggplot on our daily experimental data!